composer.json

```diff
@@ -10,6 +10,12 @@
             "name": "Asa Kusuma"
         }
     ],
+    "repositories": [
+    	{
+            "type": "vcs",
+            "url": "https://github.com/Daursu/curl"
+        }
+    ],
     "require": {
         "php": ">=5.3.0",
         "alexsoft/curl": "0.4.*@dev"
```

Comment:
No since the package name is the same "curl" composer will automatically use the new URL to clone the dependency.